process/tech-due-diligence-guidelines.md

```diff
@@ -0,0 +1,103 @@
+inclusion of each project at the relevant time.
+
+## Leading a Technical Due Diligence
+
+### Primary Goals
+
+To enable the voting TOC members to cast an informed vote about a
+project, it is crucial that each member is able to form their own
+opinion as to whether and to what extent the project meets the agreed
+upon [criteria](https://www.cncf.io/projects/graduation-criteria/) for
+inception, incubation or graduation. As the leader of a DD, your job
+is to make sure that they have whatever information they need,
+succinctly and readily available, to form that opinion.
+
+As a secondary goal, it is in the interests of the broader CNCF
+ecosystem that there exists some reasonable degree of consensus across
+the community regarding the inclusion or otherwise of projects at the
+various maturity levels.  Making sure that the relevant information is
+available, and any disagreement or misunderstanding as to it's
+validity are ideally resolved, helps to foster this consensus.
+
+### Where to start
+
+* make sure you're clear on what the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/)
+  are.  If anything's unclear to you, reach out on the TOC mailing list for advice.
+* make sure you've read, in detail, the relevant [project proposal](https://github.com/quinton-hoole/toc/tree/master/proposals).
+  This will usually be in the form of an [open pull request](https://github.com/cncf/toc/pulls).
+  I'd suggest holding off on commenting on the PR until you've completed the next three steps.
+* take a look at some [previous submissions](https://github.com/cncf/toc/pulls?utf8=%E2%9C%93&q=is%3Apr)
+  (both successful and unsuccessful) to help calibrate your expectations.
+* do as much reading up as you need to (and consult with experts in the specific field) in order to familiarize yourself with the technology
+  landscape in the immediate vicinity of the project (and don't only use the proposal and that project's documentation as a guide in this regard).
+* at this point you should have a very clear technical idea of what exactly the project actually does and does not do, roughly how it compares with and differs from 
+  similar projects in it's technology area, and/or a set of unanswered questions in those regards. 
+* go through the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/) and for each item,
+  decide for yourself whether or not you have enough info to make a strong, informed call on that item.
+  * If so, write it down, with motivation.
+  * If not, jot down what information you feel you're missing.
+  * Also take note of what unanswered questions the community might have posted in the PR review that you consider
+    to be critically important.
+    
+### Some example questions that will ideally need clear answers
+
+#### Technical
+
+* What are the current performance and scalability bounds of the software?  Have these been explicitly tested?
```

Comment:
And efficiency/resource overhead, if relevant. For example, if it is intended to run as a sidecar with every application container, it should be lightweight.

Reply:
Agreed, added.